BUILDING.md

```diff
@@ -34,6 +34,9 @@ On a bare bones installation of Debian (or a distribution derived from Debian, s
 
 ```shell
 apt-get update && apt-get install build-essential cmake curl git rustc cargo
+
+# curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs/ | sh -s -- --default-toolchain=1.76.0 -y
```

Comment:
Why is this commented out? 

Perhaps we can add an explanation here to say that if your apt doesn't install the correct version of rust, that you can then install it using this command? Or perhaps just point to the Rust documentation?

Reply:
That story is started from the point that we relied to the distribution provided Rust packages and it worked well initially.

Recently we discovered that builds will fail if Rust version < 1.76.0. And I've just updated version constrains in the guide and added a way to install a specific. It is commented out, because is considered just a helper command when distribution package does not provide a required version.

Converted it to a part of the guide.